### PR TITLE
fix: Added submit on ctrl/meta  + enter to access group dialog

### PR DESCRIPTION
**Pull request by LudoHolbik**
## Changelog description

- added access group dialog submitting form on hotkey shortcut -> ctrl/meta  + enter

### CODE REVIEW

File: src/pages/SettingsPage/AccessGroups/NewAccessGroup.jsx [hunk 1]
```diff
@@ -30,6 +30,16 @@ const NewAccessGroup = ({ onClose, accessGroupList }) => {
     error = 'This access group already exists'
   else if (!accessGroupName.match('^[a-zA-Z_]{2,20}$')) error = 'Invalid access group name'
 
+  const handleKeyDown = (e) => {
+    e?.stopPropagation()
+    const isEnter = e.key === 'Enter'
+    const isEsc = e.key === 'Escape'
+    const isCtrlMetaShift = e.ctrlKey || e.metaKey || e.shiftKey
```
Comment (Innders) [inline src/pages/SettingsPage/AccessGroups/NewAccessGroup.jsx:37]: Thanks for the quick fix. One small quick fix on top of this quick fix...

e.shiftKey is used across the app to submit a form but not to close the form modal. So you can quickly add multiple items.

To keep things consistent could we please remove e.shift or mimic this submit form but keep modal open behavior and unsure focus returns back to the input for faster creation.

**Review by Innders — CHANGES_REQUESTED** [review]

File: src/pages/SettingsPage/AccessGroups/NewAccessGroup.jsx (hunk 1 above)
Comment (LudoHolbik) [reply src/pages/SettingsPage/AccessGroups/NewAccessGroup.jsx]: I have made hotfix of hotfix 🙂 and tested it. It works as you suggested.

**Review by Innders — CHANGES_REQUESTED** [review]
ctrl and shift work great! But you can submit the form using the shortcuts even if the name is invalid/duplicate.

https://github.com/ynput/ayon-frontend/assets/49156310/2f6ea542-c8d0-45e6-ae3d-9ed2448bfeba

**Comment by LudoHolbik** [comment]
I have disabled saving if error is present (i.e. if name is duplicate or invalid format).

**Review by Innders — APPROVED** [review]

**Pull request merged**